autotests/testolmsession.cpp

```diff
@@ -45,22 +45,22 @@ void TestOlmSession::olmEncryptDecrypt()
     const auto encrypted = outboundSession->encrypt("Hello world!");
     if (encrypted.type() == QOlmMessage::PreKey) {
         QOlmMessage m(encrypted); // clone
-        QVERIFY(std::get<bool>(inboundSession->matchesInboundSession(m)));
+        QVERIFY(inboundSession->matchesInboundSession(m));
```

Comment:
IIUC this now just tests if the value is a bool, not if the bool is `true`

Reply:
No; I changed `matchesInboundSession()`. There should be no Olm error, and the value should be one.